tests/readers/test_netcdf_reader.py

```diff
@@ -221,6 +221,23 @@ def test_get_fields_missing_standard_name_attr_in_coord_array():
         assert len(fs) == 2
 
 
+def test_satellite_l2_data():
```

Comment:
Please use the same pytest decorators as for the other cds-based tests. Something like this:

```python
from earthkit.data.testing import NO_CDS


@pytest.mark.long_test
@pytest.mark.download
@pytest.mark.skipif(NO_CDS, reason="No access to CDS")
def test_satellite_l2_data():
```
Alternatively, you can add the data file to the download server and get it for the test. If it is very small it can be even added to `tests/data`.

Reply:
I added the file to the test data server. Should I include any of the other decorators?

Reply:
I added the long_test and download decorators to my new test.
I also added some decorators to the other CDS test that was in there.